nomenclature/validation_schemas/tag_schema.yaml

```diff
@@ -0,0 +1,35 @@
+$schema: 'https://json-schema.org/draft/2020-12/schema'
+title: TagList
+description: >
+  This schema is used for the validation of yaml tag files.
+type: array
+items:
+  $ref: '#/definitions/Tag'
+minItems: 1
+maxItems: 1
+
+definitions:
+
+  Tag:
+    type: object
+    patternProperties:
+      # The key of the dictionary is the tag name and must be written as `<Tag>`
+      ^<.+>$:
```

Comment:
I just had a quick look and is because of the combination of three settings in the schema in the `Tag` definition: `additionalProperties: false, minProperties: 1, maxProperties: 1`. This combination results in only one property being allowed which has to be match for the specified patternProperty. If you remove `maxProperties: 1` while keeping everything in place you can have as many keys in a tag as you want

Reply:
Perfect